src/components/gauge/gauge.tsx

```diff
@@ -0,0 +1,373 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Listen,
+  Prop,
+  State,
+  h
+} from "@stencil/core";
+
+import { Component as GxComponent } from "../common/interfaces";
+
+@Component({
+  shadow: false,
+  styleUrl: "gauge.scss",
+  tag: "gx-gauge"
+})
+export class Gauge implements GxComponent {
+  @Element() element: HTMLGxGaugeElement;
+
+  /**
+   * The `gxGaugeDidLoad` event is triggered when component has been rendered completely.
+   */
+  @Event() gxGaugeDidLoad: EventEmitter;
+
+  /**
+   * Property of type Style.
+   * Define if shadow will display or not. Default is disabled.
+   */
+  @Prop() styleShadow = false;
```

Comment:
How about just "shadow"?

Reply:
I think that comes from a previous feature/version... I will check that

Reply:
that property comes from a previous version of the control. Removed in 89570cc